### PR TITLE
Fixed the useLambdaHooks options interface

### PR DESCRIPTION
**Pull request by DavidTanner**
fixed

### CODE REVIEW

File: src/lambda.d.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import {TestInterface} from "ava";
 import Docker = require('dockerode');
 
 export interface Environment {
-  [key:string]: string | null;
```
Comment (mdlavin) [inline src/lambda.d.ts]: This change feels wrong. Doesn't the environment need to be a string or null?

Comment (DavidTanner) [reply src/lambda.d.ts]: The value should be anything that can be stringified.

Comment (DavidTanner) [reply src/lambda.d.ts]: https://github.com/lifeomic/lambda-tools/blob/master/src/lambda.js#L20

Comment (DavidTanner) [reply src/lambda.d.ts]: I can change it to match those values though.

**Pull request merged**